GG/test/acceptance/runner/Application.cpp

```diff
@@ -181,19 +181,39 @@ void MinimalGGApp::Render() {
 // application-dependent.  Note that this method is called before Render() is
 // ever called.
 void MinimalGGApp::GLInit() {
-    double ratio = Value(AppWidth() * 1.0) / Value(AppHeight());
-
     glEnable(GL_BLEND);
     glEnable(GL_CULL_FACE);
     glEnable(GL_DEPTH_TEST);
     glClearColor(0, 0, 0, 0);
     glViewport(0, 0, Value(AppWidth()), Value(AppHeight()));
     glMatrixMode(GL_PROJECTION);
     glLoadIdentity();
-    gluPerspective(50.0, ratio, 1.0, 10.0);
-    gluLookAt(0.0, 0.0, 5.0,
-              0.0, 0.0, 0.0,
-              0.0, 1.0, 0.0);
+
+    // calculate perspective with vertical FOV of 50°. 1:1 application
+    // window ratio, near plane of 1.0 and far plane of 10.0
+    float ratio = Value(AppWidth() * 1.0) / Value(AppHeight());
+    float radians = 50.0f * M_PI / 90.0f;
+    float cotangent = std::cos(radians) / std::sin(radians);
+
+    float projection[4][4] = { 0.0f };
+    projection[0][0] = cotangent / ratio;
+    projection[1][1] = cotangent;
+    projection[2][2] = -11.0f / 9.0f;
+    projection[2][3] = -1.0f;
+    projection[3][2] = -20.0f / 9.0f;
+    projection[3][3] = 0.0f;
+
+    glMultMatrixf(&projection[0][0]);
+
+    GLfloat view[4][4] = { 0.0 };
+
```

Comment:
seemingly unnecessary blank line

Reply:
Removed line and added a comment for camera placement.